0.3.1/mingw/Dockerfile

```
@@ -48,6 +48,7 @@ RUN pacman -S --noconfirm --noprogressbar \
         patch \
         ninja \
         ccache \
+		unzip \
```

Using tabs (as opposed to whitespaces above)
```suggestion
        unzip \
```